lib/rubocop/cop/bugcrowd/database.rb

```diff
@@ -37,8 +46,20 @@ def within_change_or_up_method?(node)
           )
         PATTERN
 
+        def_node_matcher :remove_index?, <<~PATTERN
+          (send nil? :remove_index ...)
+        PATTERN
+
+        def_node_matcher :remove_index_concurrently?, <<~PATTERN
+          (send nil? :remove_index _ _
+            (hash
+              <(pair (sym :algorithm) (sym :concurrently)) ...>
+            )
+          )
+        PATTERN
+
         def_node_matcher :ddl_method?, <<~PATTERN
```

Comment:
```suggestion
        def_node_matcher :requires_ddl_transaction?, <<~PATTERN
```
I like your suggestion to rename this